### ISSUE TITLE
Specify the error you receive when requesting an illegal range of bases from ListReferenceBasesRequest

### ISSUE THREAD

**Issue by hjellinek** [ga4gh/schemas#430]
For instance, if you have a reference R containing B bases and issue a `ListReferenceBasesRequest` with `start` = `0` and `end` = B + 1, what should the result be?

The Ref Server returns a GAException with HTTP status code set to `416` (`REQUESTED_RANGE_NOT_SATISFIABLE`).  This makes sense.  I will update the docs to say so.

**Comment by jeromekelleher** [comment]
Agreed. What about searchVariants and searchReads though? These both take ranges of the same form. Should we be consistent and follow this pattern for all of them (I say, yes).

**Comment by diekhans** [comment]
+1000 on keep the API behavior consistent.  Its madding when calls that appear
functional equivalent behave differently
